rust/lance-core/Cargo.toml

```diff
@@ -60,6 +60,7 @@ rand.workspace = true
 tempfile.workspace = true
 lance-testing.workspace = true
 parquet.workspace = true
+proptest = "1.0.0"
```

Comment:
I guess I can specify the current version, in case there are newer features.

In case you didn't know, in Cargo when you specify `proptest = "1.3.1"` it doesn't mean like in python `proptest==1.3.1`, it means `proptest>=1.3.1<2.0.0`. https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-cratesio

Reply:
I did not know that. Thanks for letting me know! :D